include/psa/crypto.h

```diff
@@ -1552,7 +1663,12 @@ psa_status_t psa_cipher_encrypt(psa_key_handle_t handle,
  * \retval #PSA_ERROR_INSUFFICIENT_MEMORY
  * \retval #PSA_ERROR_COMMUNICATION_FAILURE
  * \retval #PSA_ERROR_HARDWARE_FAILURE
+ * \retval #PSA_ERROR_STORAGE_FAILURE
  * \retval #PSA_ERROR_CORRUPTION_DETECTED
```

Comment:
Order of these errors should probably match betweeen `psa_cipher_encrypt()` and `psa_cipher_decrypt()`. Not a blocker.